release-notes/elastic-observability/index.md

```diff
@@ -21,6 +21,62 @@ To check for security updates, go to [Security announcements for the Elastic sta
 % ### Fixes [elastic-observability-next-fixes]
 % *
 
+## 9.2.0 [elastic-observability-9.2.0-release-notes]
+
+### Features and enhancements[elastic-observability-9.2.0-features]
+
+* users can create routing conditions directly from preview table in Streams. [#235560]({{kib-pull}}235560).
+* Adds new Observability navigation [#233784]({{kib-pull}}233784).
+* Observability navigation uses new icons [#236808]({{kib-pull}}236808).
+* Adds functional tests for Logs Essentials tier to cover tier-specific behavior [#234904]({{kib-pull}}234904).
+* Observability onboarding Playwright tests now support Logs Essentials tier [#234644]({{kib-pull}}234644).
+* Adds **View in Discover** to the alerts details page for Synthetics Monitor Status and TLS alert rules. [#234104]({{kib-pull}}234104).
+* Adds **View in discover** to the alerts details page for SLO burn rate and ES query rules [#233855]({{kib-pull}}233855).
+* Adds **View in discover** button in alert details header [#233259]({{kib-pull}}233259).
+* Adds time range filter to links from alert details to related dashboards [#230601]({{kib-pull}}230601).
+* Adds rules callout in metric, logs, and inventory rules [#237085]({{kib-pull}}237085).
+* Adds ability for users choose how to recover alerts, when the monitor is back up or when the condition is no longer met. [#229962]({{kib-pull}}229962).
+* Adds the **Trace timeline** to Discover flyout [#234072]({{kib-pull}}234072).
+* Adds `send_traces`, `send_metrics`, and `send_logs` agent configuration settings for EDOT Node.js [#233798]({{kib-pull}}233798).
+* Adds the ability to link dashboards to SLOs [#233265]({{kib-pull}}233265).
+* Adds dashboard references to SLO saved objects [#232583]({{kib-pull}}232583).
+* Adds dashboard tags for linked dashboards and suggested dashboards on the alert details page **Related dashboards** tab [#228902]({{kib-pull}}228902).
+* When fetching dashboards, referenced panels are also fetched [#228811]({{kib-pull}}228811).
+* Moves the installation settings for the Knowledge Base and Product Docs components into a single location. [#232559]({{kib-pull}}232559).
+* Observability AI Assistant is now aware of LLM-facing documentation for any installed integrations in the user's cluster [#237085]({{kib-pull}}237085).
+* Adds `raw_request` to traces for `.gen-ai`, `.gemini`, and `.bedrock` connectors [#232229]({{kib-pull}}232229).
+* Installs product docs when the knowledge base is installed [#228695]({{kib-pull}}228695).
+* Adds manual test for bulk import functionality for Observability AI Assistant [#225497]({{kib-pull}}225497).
+* Improves the AI Assistant Settings page by updating the logos to be solution-specific [#224906]({{kib-pull}}224906).
```

Comment:
Not sure whether you want to call it Observability AI Assistant or AI Assistant throughout here, but probably best to pick one. My vote would be for "AI Assistant" for simplicity